Gems/Profiler/Code/Source/CpuProfiler.h

```diff
@@ -227,5 +227,6 @@ namespace Profiler
         CpuProfilingStatisticsSerializer(const AZStd::ring_buffer<TimeRegionMap>& continuousData);
 
         AZStd::vector<CpuProfilingStatisticsSerializerEntry> m_cpuProfilingStatisticsSerializerEntries;
+        AZStd::sys_time_t m_timeTicksPerSecond = 0;
```

Comment:
It is simpler to just store an [AZStd::chrono::microseconds](https://en.cppreference.com/w/cpp/chrono/duration)

Reply:
```suggestion
        AZStd::chrono::microseconds m_timeMicroseconds = 0;
```